### PR TITLE
SimonasB88 - Taxes

### PR DESCRIPTION
**Pull request by SimonasB88**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

**Review by MatShir — CHANGES_REQUESTED** [review]

File: content/1.7/back-office/international/taxes/add-edit-taxes.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Top elements of Add or Edit Taxes
+
+There is an information notification with the text _Note that this feature is available in all shops context only. It will be added to all your stores_ meaning that tax editing will be aligned the same with the all multistores and shop groups.
+
+## Tax Editing form
+
+- **Name** - mandatory input field for naming the specific Tax
+- **Language dropdown** - enables to trigger the appropriate language for the Name of tax. There can different tax names set for different languages.
+- **Rate** - numeric input field, allows setting the Range of the tax. Range must be typed as a single numbers or single numbers with decimals.
```
[the remark this answers is not in the document]
Comment (SimonasB88) [reply content/1.7/back-office/international/taxes/add-edit-taxes.md:26]: oops

File: content/1.7/back-office/international/taxes/add-edit-taxes.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Notifications after submissions
+
+If there is an empty Range field after submission, there will be a browser error notification, requesting to fill the empty field.
+
+If there are some invalid characters typed in name after submission, there will be an error below the Name field - _"{invalid character}" is invalid - Language: {the language of the name}_.
+
+If the Name field is left empty after submission, there will be an error below the Name field - _The field name is required at least in your default language._
+
+If the Range is typed with more than 6 characters, the UI will be prompted with the error - _This field cannot be longer than 6 characters_.
```
[the remark this answers is not in the document]
Comment (SimonasB88) [reply content/1.7/back-office/international/taxes/add-edit-taxes.md:35]: oops

File: content/1.7/back-office/international/taxes/taxes-listing.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  - 50
+  - 100
+
+## Tax options table
+
+- **Enable tax** - toggle switch button, enabling or disabling Tax showing and calculations on the market in Prestashop items. By default it is enabled.
+- **Display tax in the shopping cart** - toggle switch button, enabling or disabling Tax showing in the Cart. By default it is disabled.
+- **Based on** - dropdown element to choose from where and what location should the Taxes calculate. Possible values are **Invoice Address** and **Delivery Address**. By default, there is Invoice address value selected.
+- **Use ecotax** - toggle switch button, controlling the ecotaxes availability. Enabling the ecotaxes, creates additional field in Pricing area in Product Page editing. By default, the ecotaxes are disabled.
+- **Ecotax** - if _Use ecotax_ is enabled, this _Ecotax_ dropdown field appears. There can be certain ecotax and range set from the dropdown. This field becomes mandatory.
```
[the remark this answers is not in the document]
Comment (SimonasB88) [reply content/1.7/back-office/international/taxes/taxes-listing.md:67]: Yes, the eco tax rate is displayed, and the price is adjusted also in Product Page.

[the remark this answers is not in the document]
Comment (MatShir) [reply content/1.7/back-office/international/taxes/taxes-listing.md:67]: Back Office / Front Office ?

[the remark this answers is not in the document]
Comment (SimonasB88) [reply content/1.7/back-office/international/taxes/taxes-listing.md:67]: Yea, both BO and FO, will update the specs

File: content/1.7/back-office/international/taxes/taxes-listing.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+- **Viewing indicator text** - text, indicating what pages the user is seeing.
+- **Items per page** - dropdown with the numeric values, how many items can be shown in listing:
+  - 10
+  - 20
+  - 50
+  - 100
+
+## Tax options table
+
+- **Enable tax** - toggle switch button, enabling or disabling Tax showing and calculations on the market in Prestashop items. By default it is enabled.
```
[the remark this answers is not in the document]
Comment (SimonasB88) [reply content/1.7/back-office/international/taxes/taxes-listing.md]: @MatShir 
Tested those cases - I can not detect any difference or incorrect displaying in back-office, product page or order page.

File: content/1.7/back-office/international/taxes/taxes-listing.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+- **Help** - a side-bar with the helpful information, how to navigate in this page.
+- **Tabs: Taxes and Tax Rules** - navigation tabs for taxes and tax rules.
+
+## Middle elements - Tax table
+
+- **Taxes** - table title with the total taxes in the list.
+- **Numeric value** - total sum of shopping taxes identified.
+- **Export icon** - when clicked, a file is generated with all **displayed** taxes from the data list in CSV format.
+- **Refresh** - once clicked, the whole page reloads to display the latest data.
+
```
Comment (camilleganzin) [inline content/1.7/back-office/international/taxes/taxes-listing.md:24]: - **Wheel icon** - when clicked, these actions are available: 
    - **Export icon** - when clicked, a file is generated with all **displayed** taxes from the data list in CSV format.
    - **Refresh** - once clicked, the whole page reloads to display the latest data.

**Review by camilleganzin — CHANGES_REQUESTED** [review]

File: content/1.7/back-office/international/taxes/international-tax-rules-listing.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+- **Recommended Modules and Services** - redirects to recommended modules and services page.
+- **Help** - a side-bar with the helpful information, how to navigate in this page.
+- **Tabs: Taxes and Tax Rules** - navigation tabs for taxes and tax rules.
+
+## Middle elements - Tax Rules table
+
+- **Taxes** - table title with the total tax rules in the list.
+- **Numeric value** - total sum of shopping tax rules identified.
+- **Export icon** - when clicked, a file is generated with all **displayed** tax rules from the data list in CSV format.
+- **Refresh** - once clicked, the whole page reloads to display the latest data.
```
[the remark this answers is not in the document]
Comment (SimonasB88) [reply content/1.7/back-office/international/taxes/international-tax-rules-listing.md:25]: @camilleganzin makes sense, thanks.

File: content/1.7/back-office/international/taxes/international-tax-rules-listing.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+### Filtering
+
+- **ID** - input field, for searching and filtering specific Tax Rule by ID.
+- **Name** - input field, for searching and filtering specific Tax Rule by name.
+- **Enabled** - dropdown element, for arranging the Tax Rules by enabled or disabled.
+- **Search button** - executes the filtering action, shows the results.
+- **Reset** - resets the filtering, shows all the unfiltered Tax Rules.
```
[the remark this answers is not in the document]
Comment (SimonasB88) [reply content/1.7/back-office/international/taxes/international-tax-rules-listing.md:35]: @camilleganzin ok, makes sense, thanks.

File: content/1.7/back-office/international/taxes/taxes-listing.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+### Filtering
+
+- **ID** - input field, for searching and filtering specific Tax by ID.
+- **Name** - input field, for searching and filtering specific Tax by name.
+- **Rate** - input field, for searching and filtering specific Tax by rate.
+- **Enabled** - dropdown element, for arranging the Taxes by enabled or disabled.
+- **Search button** - executes the filtering action, shows the results.
+- **Reset** - resets the filtering, shows all the unfiltered Taxes.
+
```
[the remark this answers is not in the document]
Comment (SimonasB88) [reply content/1.7/back-office/international/taxes/taxes-listing.md:41]: @camilleganzin yep, makes sense.

File: content/1.7/back-office/international/taxes/taxes-listing.md (hunk 5 above)
Comment (SimonasB88) [reply content/1.7/back-office/international/taxes/taxes-listing.md:24]: @camilleganzin could we specify as Gear icon? It is more like that, I guess. And there are 3 values in dropdown on PS1780 
image

So, what could be better? 🙂

**Review by marwachelly — COMMENTED** [review]
Hello @SimonasB88,

Could you please check the requested changes? 

Thanks,

File: content/1.7/back-office/international/taxes/add-edit-tax-rules.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+# INTERNATIONAL > Taxes > Tax Rules > Add or Edit Tax Rule
+
+![Adding or Editing Tax Rules](static/img/international-add-edit-tax-rules.png)
+
+This page UI allows to edit existing or add a brand new Tax Rule to the Prestashop Tax system.
+
+## Top elements
+
+- **Add New or Edit:{edited Tax Rule name}** - page title.
+- **Add new tax rule CTA button** - toggles the expanding or subtracting the Tax rule creating and editing section below.
```
[the remark this answers is not in the document]
Comment (SimonasB88) [reply content/1.7/back-office/international/taxes/add-edit-tax-rules.md:15]: @marwachelly One is just title and the other component - CTA button 
image

File: content/1.7/back-office/international/taxes/taxes-listing.md (hunk 5 above)
Comment (SimonasB88) [reply content/1.7/back-office/international/taxes/taxes-listing.md:24]: @camilleganzin @marwachelly hey, maybe you have insights how I should update correctly? 🙂 Thanks.

**Pull request merged**